applications/FluidDynamicsApplication/custom_strategies/schemes/simple_steady_adjoint_scheme.h

```diff
@@ -66,15 +66,16 @@ class SimpleSteadyAdjointScheme : public ResidualBasedAdjointStaticScheme<TSpars
     /// Constructor.
     explicit SimpleSteadyAdjointScheme(
         AdjointResponseFunction::Pointer pResponseFunction,
+        const IndexType Dimension,
         const IndexType BlockSize)
```

Comment:
```suggestion
        const IndexType BlockSize = Dimension + 1)
```
(minor suggestion, feel free to leave it as it is)

Reply:
I was intentionally doing like that because, I had a nasty mistake when I copy pasted same stuff from python fluid to RANS, but forgot to change the block size. It was not complaining, giving wrong results (Took me a while to identify the problem :/)